recipes-devtools/llvm/llvm_%.bbappend

```diff
@@ -0,0 +1,2 @@
+# Temporarily until a corresponding change is merged to OE-Core
```

Comment:
Since this is temporary, it would be preferable to use the specific version `llvm_20.1.0.bbappend` so that this change is not forgotten here.

Reply:
I tried doing this. Unfortunately, it makes handling of this PR much more complicated as the Yoe is not in full sync with OE-Core. I tried pushing a correct versioned bbappend and the build failed. Anyway, I've opened #1076 as a draft, so that we don't forget to drop it.

Reply:
yeah the draft is a good reminder.